src/Robo/Commands/Acsf/AcsfCommand.php

```diff
@@ -34,13 +34,11 @@ public function acsfInitialize($options = ['acsf-version' => '^1.33.0']) {
    * @command acsf:init:drush
    */
   public function acsfDrushInitialize() {
-    $drushBin = $this->getConfigValue('drush.bin');
-
     $this->say('Executing initialization command for acsf module.');
 
-    $this->taskExec("{$drushBin} acsf-init --include={$this->getConfigValue('docroot')}/modules/contrib/acsf/acsf_init --yes")
-      ->printOutput(TRUE)
-      ->dir($this->getConfigValue('docroot'))
+    $this->taskDrush()
+      ->option("--include={$this->getConfigValue('docroot')}/modules/contrib/acsf/acsf_init")
```

Comment:
I agree, it's odd defining the option before the command. I'll refactor in a subsequent PR (unless you feel like it's a blocker - which I would understand).